src/gui/MItem_menus.cpp

```diff
@@ -402,17 +402,32 @@ MI_LOAD_SETTINGS::MI_LOAD_SETTINGS()
     : WI_LABEL_t(_(label), nullptr, is_enabled_t::yes, is_hidden_t::no) {
 }
 
+static void add_load_response(const char *name, bool flag, char *msg, int msglen) {
+    char tmp[strlen(name) + 5];
+    snprintf(tmp, sizeof(tmp), "%s: %s\n", name, flag ? "+" : "-");
+    strncat(msg, tmp, msglen);
+}
+
 void MI_LOAD_SETTINGS::click(IWindowMenu & /*window_menu*/) {
-    // FIXME: Some error handling/reporting
-    // TODO: Loading other things than just network
+    bool is_network = false;
+    bool is_connect = false;
+
     if (netdev_load_ini_to_eeprom()) {
         notify_reconfigure();
+        is_network = true;
     }
 
-// FIXME: Error handling
 #if BUDDY_ENABLE_CONNECT()
-    connect_client::MarlinPrinter::load_cfg_from_ini();
+    if (connect_client::MarlinPrinter::load_cfg_from_ini()) {
+        is_connect = true;
+    }
+#endif
+    char msg[MSG_MAX_LENGTH] = "\nLoading settings finished.\n\n";
```

Comment:
Why is there a newline at the beginning of the text?

Reply:
That's for optical reasons, looks better this way.